starter-code/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Cluedo game</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" media="screen" href="css/style.css" />
+
+</head>
+
+<body>
+  <div class="container">
+    <div class="overview">
+
+      <!-- character IMG -->
+      <div class="block">
+        <img id="character" alt="Murderer">
+      </div>
+
+      <!-- character details -->
+      <div class="block">
+        <h2>Murderer details</h2>
+        <ul>
+          <li id="character-firstname"></li>
+          <li id="character-lastname"></li>
+          <li id="character-age"></li>
+          <li id="character-profession"></li>
+          <li id="character-description"></li>
+        </ul>
```

Comment:
Generally speaking, if you have to repeat a word several times, it should probably be a class.  If i were you, I would make `character`, `weapon` and `room` classes.    then you could give your block an additional class `character`:
```
  <div class="block character">
        <h2>Murderer details</h2>
        <ul>
          <li id="firstname"></li>
          <li id="lastname"></li>
          <li id="age"></li>
          <li id="profession"></li>
          <li id="description"></li>
        </ul>
```
And then in your JS you could do something like: 
```
var characterName = document.querySelector(".character #firstname")
```

Reply:
[List of JS DOM Methods](https://www.w3schools.com/jsref/dom_obj_document.asp)